docs/src/visualizing.md

```diff
@@ -2,10 +2,10 @@ All plotting functionality of `DynamicalBilliards` lies within a few well-define
 
 All plotting functions are brought into scope when `using PyPlot`. The functions are:
 ```@docs
-plot_obstacle
-plot_particle
+plot
+plot
 plot_cyclotron
-plot_billiard
+plot
 animate_evolution
 ```
```

Comment:
Here is the problem for the warning you are seeing: Remove all the extra plots (leave only one line with `plot` and instead of just `plot` do instead `DynamicalBilliards.plot`.